Big payload cause error while displaying report

```
2018-01-22 14:48:30,430 [ERROR] from application in application-akka.actor.default-dispatcher-37 -
2018-01-22T15:48:30.876+01:00! @76ldmpmfo - Internal server error, for (GET) [/api/experiments/izanami:example:button/results] ->
2018-01-22T15:48:30.877+01:00%{syslog_message}
2018-01-22T15:48:30.877+01:00play.api.UnexpectedException: Unexpected exception[EntityStreamSizeException: EntityStreamSizeException: actual entity size (Some(14717622)) exceeded content length limit (8388608 bytes)! You can configure this by setting `akka.http.[server|client].parsing.max-content-length` or calling `HttpEntity.withSizeLimit` before materializing the dataBytes stream.]
```

--- a/izanami-server/app/domains/abtesting/impl/ExperimentVariantEventElasticStore.scala
+++ b/izanami-server/app/domains/abtesting/impl/ExperimentVariantEventElasticStore.scala
@@ -230,8 +230,8 @@ class ExperimentVariantEventElasticStore(client: Elastic[JsValue],
               "query" -> Json.obj(
                 "bool" -> Json.obj(
                   "must" -> Json.arr(
-                    Json.obj("term" -> Json.obj("id"        -> experiment.id.key)),
-                    Json.obj("term" -> Json.obj("variantId" -> v.id))
+                    Json.obj("term" -> Json.obj("experimentId" -> experiment.id.key)),
+                    Json.obj("term" -> Json.obj("variantId"    -> v.id))
                   )
                 )
               )
@@ -244,7 +244,7 @@ class ExperimentVariantEventElasticStore(client: Elastic[JsValue],
       .map { id =>
         Bulk[ExperimentVariantEvent](BulkOpType(delete = Some(BulkOpDetail(None, None, Some(id)))), None)
       }
-      .via(client.bulkFlow(batchSize = 500))
+      .via(index.bulkFlow(batchSize = 500))
       .runWith(Sink.ignore)
       .map(_ => Result.ok(Done))
 
@@ -287,7 +287,7 @@ class ExperimentVariantEventElasticStore(client: Elastic[JsValue],
     val query = Json.obj(
       "size"    -> 1,
       "_source" -> Json.arr("date"),
-      "query"   -> Json.obj("term" -> Json.obj("id" -> Json.obj("value" -> experimentId))),
+      "query"   -> Json.obj("term" -> Json.obj("experimentId" -> Json.obj("value" -> experimentId))),
       "sort"    -> Json.arr(Json.obj("date" -> Json.obj("order" -> order)))
     )
     Logger.debug(s"Querying ${Json.prettyPrint(query)}")
@@ -310,11 +310,10 @@ class ExperimentVariantEventElasticStore(client: Elastic[JsValue],
     import cats.instances.future._
 
     val minDate: Future[Option[LocalDateTime]] = min(experimentId)
-    val maxDate: Future[Option[LocalDateTime]] = max(experimentId)
 
     (for {
       min <- OptionT(minDate)
-      max <- OptionT(maxDate)
+      max = LocalDateTime.now()
     } yield {
       if (ChronoUnit.MONTHS.between(min, max) > 50) {
         "month"

